Initial manifest for com.moonlight_stream.Moonlight

cgutman opened the pull request:
Submitting this as the upstream maintainer for https://github.com/moonlight-stream/moonlight-qt. We'd like to use Flathub and Flatpaks as our primary method of app distribution for Linux.

nedrichards commented:
bot, build com.moonlight_stream.Moonlight

nedrichards commented on com.moonlight_stream.Moonlight.json:
```diff
@@ -0,0 +1,50 @@
+	"runtime-version": "5.11",
+	"sdk": "org.kde.Sdk",
+	"command": "moonlight",
+	"rename-icon": "moonlight",
```
not a blocker, but if flatpak is your primary then it's normally better to have your icons with exportable names rather than rename them (saves accidentally missing assets if you hard code it somewhere)

cgutman replied:
yeah, I did that with the other resources. The icon was more problematic because that same SVG is actually bundled into the app via QRC file for all platforms and used to set the window icon at runtime.

nedrichards commented on com.moonlight_stream.Moonlight.json:
```diff
@@ -0,0 +1,50 @@
+		"--socket=x11",
+		"--share=ipc",
+		"--socket=pulseaudio",
+		"--device=all"
```
I assume this is for input devices.

cgutman replied:
yep

cgutman commented:
Looks like the NEON stuff puked on ARMv7. I think I'll remove the libopus Git submodule and QMake project, then just include libopus as a module like I did with ffmpeg. That way I can use the existing build system for libopus.

cgutman commented:
It turns out libopus is already included in the KDE platform, so I didn't need to add it as a module after all. I've removed the custom QMake project and switched to the platform's libopus to address the ARMv7 build failure.

This should be ready for a build and review again.

nedrichards commented:
bot, build com.moonlight_stream.Moonlight
